apps/src/lab2/types.ts

```diff
@@ -213,6 +213,7 @@ export interface LevelProperties {
   startDirection?: number;
   widgetView?: boolean;
   widgetViewAllowShowCode?: boolean;
+  aiTutor2Available?: boolean;
```

Comment:
Is this `aiTutor2Available` because it's the 2nd evolution of AI Tutor and you just need a way to keep them separate as you build? or is the distinction between the hint panel and the general chat panel? If it's the latter, I'd vote for something like `aiTutorHintAvailable` and `aiTutorChatAvailable`

Reply:
oops - I kept reading 😁 it's just to keep things straight in this weird interim world